wye/templates/about.html

```diff
@@ -2,7 +2,7 @@
 {% block title %}About | Pythonexpress{% endblock %}
 {% block content %}
 
-<p>We are the team of volunteers who wanted to teach python and create awareness about Python programming Language</p>
+<p>We are the team of volunteers who are willing to teach python and create awareness about Python programming Language</p>
```

Comment:
I think we need to add more details about us
1) We are experienced people who have industry and real time python knowledge?
2)Why are we conducting free workshop?
and etc.